astropy/modeling/tabular.py

```diff
@@ -190,4 +219,62 @@ def tabular_model(dim, name=None):
 
 
 Tabular1D = tabular_model(1, name='Tabular1D')
+
 Tabular2D = tabular_model(2, name='Tabular2D')
+
+_tab_docs = """
+    method : str, optional
+        The method of interpolation to perform. Supported are "linear" and
+        "nearest", and "splinef2d". "splinef2d" is only supported for
+        2-dimensional data. Default is "linear".
+    bounds_error : bool, optional
+        If True, when interpolated values are requested outside of the
+        domain of the input data, a ValueError is raised.
+        If False, then `fill_value` is used.
+    fill_value : float, optional
+        If provided, the value to use for points outside of the
+        interpolation domain. If None, values outside
+        the domain are extrapolated.  Extrapolation is not supported by method
+        "splinef2d".
+
+    Returns
+    -------
+    value : ndarray
+        Interpolated values at input coordinates.
+
+    Raises
+    ------
+    ImportError
+        Scipy is not installed.
+
+    Notes
+    -----
+    Uses `scipy.interpolate.interpn`.
+"""
+
+
+Tabular1D.__doc__  =  """
```

Comment:
For some reason, the Sphinx test on Travis is not picking this up. Does it work when you compile the doc locally?

Also, why not also use `+ _tab_docs` for `Tabular` base class itself?